packages/app/src/cli/services/flow/constants.ts

```diff
@@ -30,7 +30,7 @@ export const uiTypesMap: [string, string][] = [
   ['boolean', 'checkbox'],
   ['email', 'email'],
   ['multi_line_text_field', 'text-multi-lines'],
-  ['number_integer', 'int'],
```

Comment:
Based off on [this](https://github.com/Shopify/flow/blob/main/lib/flow_core/application/connectors.rb#L938) it seemed to me like `number_integer` should be `int` 🤔